backend/services/interfaces/statisticsService.ts

```diff
@@ -0,0 +1,36 @@
+import { SchoolResponseDTO } from "./schoolService";
+import { TestResponseDTO } from "./testService";
+
+export interface AggregateTestSessionDTO {
+  /** the ID of the corresponding test from the Test collection */
+  test: string;
+  /** the ID of the corresponding school from the School collection */
+  id: string;
+  /** the average overall test score */
+  averageScore: number;
+  /** the array of test result breakdowns */
+  resultsPerTestArr: number[][];
+  /** the array of average test scores for each Question */
+  averageScoresByQuestions: number[];
+}
+
+export interface StatisticsResponseDTO {
```

Comment:
Should we name this something more specific? Since we will have this type of DTO for every category that we breakdown by (i.e school, sub-region, country)

Reply:
yes i was thinking that.. also wanted to see how others would implement (e.g. we could have a "filter/grouping" field instead that takes in either a school, sub-region or country). but might be more clear to keep them separate.

Reply:
will make separate one for now! can refactor later if needed

Reply:
actually, i'll refactor AggregateTestSessionDTO to be reusable and we can have separate response dtos